Tiltfile

```diff
@@ -245,7 +245,7 @@ def capz():
     # Set up an image build for the provider. The live update configuration syncs the output from the local_resource
     # build into the container.
     docker_build(
-        ref = "gcr.io/k8s-staging-cluster-api-azure/cluster-api-azure-controller",
+        ref = "nhkregistry.azurecr.io/cluster-api-azure-controller",
```

Comment:
Will restore this `gcr.io/k8s-staging-cluster-api-azure/cluster-api-azure-controller`